Doc/tools/susp-ignored.csv

```diff
@@ -110,6 +110,7 @@ howto/pyporting,,::,Programming Language :: Python :: 3
 howto/regex,,::,
 howto/regex,,:foo,(?:foo)
 howto/urllib2,,:password,"""joe:password@example.com"""
+library/__main__,,`,
```

Comment:
Is this to ignore backticks in output on line https://github.com/python/cpython/pull/26883/files#diff-ada229cf79d40b37dc14942330de96f7d48f2019cf0b1c864e8644765a4e3e7eR319 ?  or somewhere else?

Reply:
yes